assets/js/pages/MapScreen.js

```diff
@@ -46,14 +60,19 @@ const styles = {
     boxShadow: "0px 2px 4px 0px rgba(0,0,0,0.5)",
     cursor: "pointer",
   },
+  mapStyle: {
+    version: 8,
+    sources: {
+      protomaps: {
+        type: "vector",
+        tiles: [`https://pmtiles.heliumfoundation.wtf/world/{z}/{x}/{y}.mvt`],
```

Comment:
Should we env var this in and default to the public protomaps-hosted service?

Reply:
we already have it exposed in network explorer and its by no means a secret which is why I opted to just include it. Similar to how we were already including a mapbox key beforehand for ease of use.